terms.html

```diff
@@ -5,12 +5,18 @@
 <dl class="termlist">
   <dt><dfn data-lt="claims">claim</dfn></dt>
   <dd>
-An assertion made about a <a>subject</a>.
+A digital representation of an assertion made about an <a>entity</a> by a specific <a>entity</a>.
```

Comment:
define or cite definition for "assertion" ?

Reply:
Assertion: the act of asserting, or something that is asserted. See https://www.merriam-webster.com/dictionary/assertion for details. Perhaps we should say, as ISO does, that language that isn't specifically defined in expected to mean what it says from dictionary X. ISO uses the Oxford English Dictionary (OED). As W3C is predominantly American, we could use Merriam-Webster.

Reply:
Assertions can be made about any kind of subject. Subjects need not be entities. They can be concepts.

Reply:
@jandrieu: In the current spec on the Internet, a 'claim' is 'an assertion made about a subject', and 'subject' is defined as 'a thing about which claims are made'. This is not only a cyclic definition (which I think is a bad practice), but it implies that claims can only be about things, and not about non-things.

I would argue that claims can be made about anything that is known to exist, and my guess is that @jandrieu (and others) would agree. It can be a person (yourself included), a computer, an issuer, a holder, a verifier, an extinct animal, a thought, an idea, a concept, a JSON-object, a verifiable credential, a claim, etc. Really, it is _anything_ you can think of/know to exist.

Since the term 'entity' is already explicitly defined as something different than what we apparently need here, and since the term 'thing' doesn't (really) cut it, I would say we are in need of a term that means 'anything that is known to exist', and that could be defined, e.g., as "anything that is known to exist, e.g., a person (yourself included), a computer, an issuer, a holder, a verifier, an extinct animal, a thought, an idea, a concept, a JSON-object, a verifiable credential, a claim, etc. Really, it is _anything_ you can think of/know to exist." 

We can then use this term to define 'claim' as "A digital representation of an assertion made about a <a>{{fill in the chosen term here}}</a> by a specific <a>entity</a>."

I'm curious what term we will end up with.

Reply:
```suggestion
A digital representation of an assertion made by a specific <a>entity</a>. Anything can be the <a>subject</a> of a claim.
```

Reply:
I've added suggested text that I believe may resolve this https://github.com/w3c/vc-data-model/pull/1172/files#r1282231464

Reply:
> I think there are assertions that should not qualify as a claim

Do you propose to review the content of every generated VC in order to confirm that the claims contained therein all "qualify" as such in your view? This goes far into the "impossible" category.

Reply:
No. I'm fine with claims that have a subject (identifier) and a predicate (identifier) and optionally, an object (identifier).